lib/inline-imports.js

```diff
@@ -66,13 +62,28 @@ module.exports = function inlineImports(fileContents, options, visited = new Set
         },
         visited,
       );
-
-      result.push(inlined);
-
-      files[resolvedPath] = true;
+      callback(inlined, true, i + 1);
     } else {
-      result.push(line);
+      callback(line, false, i + 1);
     }
   }
+}
+
+function inlineImports(fileContents, options, visited = new Set()) {
+  const result = [];
+  _inlineImports(fileContents, options, visited, function (line) {
```

Comment:
_inlineImports with a callback with an argument of `line` is odd, if bellow the arguments are `imports, match, lineNumber` could we change this to be named consistently?

Reply:
👍